docs/source/deployorderer/ordererdeploy.md

```diff
@@ -0,0 +1,299 @@
+
+After you have completed editing the `configtx.yaml` to reflect the orderer and peer organizations that will participate in your network, run the following command to create the genesis block of the system channel:
+```
+configtxgen -profile TwoOrgsOrdererGenesis -channelID system-channel -outputBlock ./system-genesis-block/genesis.block
+```
+
+Where:
+- `-profile` refers to the `TwoOrgsOrdererGenesis` profile in `configtx.yaml`.
+- `-channelID` must be `system-channel`.
+- `-outputCreateChannelTx` refers to the location of the generated genesis block.
+
+When the command is successful, you will see logs of `configtxgen` loading the `configtx.yaml` file and printing a channel creation transaction:
+```
+INFO 001 Loading configuration
+INFO 002 Loaded configuration: /Usrs/fabric-samples/test-network/configtx/configtx.yaml
+INFO 003 Generating new channel configtx
+INFO 004 Writing new channel tx
+```
+
+Make note of the generated `.pb` file. This is your genesis block and will be referenced in the `orderer.yaml` file below.
+
+### Storage
+
+You must provision persistent storage for your ledgers. The default location for the ledger is located at `/var/hyperledger/production/orderer`. Ensure that your orderer has write access to the folder. If you choose to use a different location, provide that path in the `FileLedger:` parameter in the `orderer.yaml` file. If you decide to use Kubernetes or Docker, recall that in a containerized environment, local storage disappears when the container goes away, so you will need to provision or mount persistent storage for the ledger before you deploy an orderer.
+
+### Configuration of `orderer.yaml`
+
+Now you can use the [Checklist for a production orderer](./ordererchecklist.html) to modify the default settings in the `orderer.yaml` file. In the future, if you decide to deploy the orderer through Kubernetes or Docker, you can override the same default settings by using environment variables instead. Check out the [note](../deployment_guide_overview.html#step-five-deploy-orderers-and-ordering-nodes) in the deployment guide overview for instructions on how to construct the environment variable names for an override.
+
+At a minimum, you need to configure the following parameters:
+- `General.ListenAddress` - Hostname that the ordering node listens on.
+- `General.ListenPort` - Port that the ordering node listens on.
+- `General.TLS.Enabled: true` - Server-side TLS should be enabled in all production networks.
+- `General.TLS.PrivateKey ` - Ordering node private key from TLS CA.
+- `General.TLS.Certificate ` - Ordering node signed certificate (public key) from the TLS CA.
+- `General.TLS.RootCAS` - This value should be unset.
+- `General.Cluster.ClientCertificate` - Ordering node signed certificate (public key) from the TLS CA.
+- `General.Cluster.ClientPrivateKey` - Ordering node private key from TLS CA.
```

Comment:
These two are optional as of v2.3 since they now default based on General.TLS.Certificate and General.TLS.PrivateKey

Reply:
Since this will be ported back to v2.2 I think we wanted to make another pass with updates like this for v2.3 after this gets merged for 2.2

Reply:
Although I did ask for this fix to be ported back to v2.2. Let me check if it made it in.

Reply:
It was backported [here](https://github.com/hyperledger/fabric/pull/2078#issuecomment-726768203) so we can add it

Reply:
Which too? Was this added and I missed it? @pamandrejko

Reply:
We have covered this in the checklist where we say:
As of Fabric v2.3.0, if unset, the `ClientCertificate` and `ClientPrivateKey` default to the server `General.TLS.Certificate` and `General.TLS.PrivateKey` when the orderer is not configured to use a separate cluster port.

Therefore I think we can remove the following two parameters from here since it would not be a typical use case to specify them.
- `General.Cluster.ClientCertificate` - Ordering node signed certificate (public key) from the TLS CA.
- `General.Cluster.ClientPrivateKey` - Ordering node private key from TLS CA.